shopinvader_delivery_note/models/sale_order.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
```

Comment:
@acsonefho IMO this code and the one into shopinvader_delivery_note should go into an new OCA addons: sale_delivery_note. It's not the intend of shopinvader to provide custom functionalities. Shopinvader must remains a service layer on top of business addons...